src/olympia/reviewers/utils.py

```diff
@@ -657,12 +657,7 @@ def get_actions(self):
             'method': self.handler.reject_multiple_versions,
             'label': 'Reject Multiple Versions',
             'minimal': True,
-            'delayable': (
-                # Either the version is listed
-                not version_is_unlisted
-                # or (unlisted and) awaiting review
-                or self.version.file.status == amo.STATUS_AWAITING_REVIEW
-            ),
+            'delayable': True,
```

Comment:
The PR title says enable delayed rejections for unlisted versions, and the change seems to enable globally. I'm guessing The goal here is to enable delayed rejections for "all" versions.  We no longer care about a file status of "STATUS_AWAITING_REVIEW"?

Reply:
This change removes the condition to only allow delay-rejecting pending unlisted versions (on top of listed versions), so that all unlisted versions can be delay-rejected.